### PR TITLE
Added ``excludes`` and extra filters (or, and)

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/dashboards/filter_parser.py [hunk 1]
```diff
@@ -1,31 +1,52 @@
...
 
-    def __init__(self, queryset, filters):
-        self.queryset = queryset
+    def __init__(self, queryset, filters, exclude=False):
         self.filters = filters
+        if exclude:
+            self.queryset_func = queryset.exclude
+        else:
+            self.queryset_func = queryset.filter
+        Q(a=1)
```
Comment (szok) [inline src/ralph/dashboards/filter_parser.py]: What it is?

File: docs/user/dashboards.md [hunk 2]
```diff
@@ -35,6 +36,19 @@ Final result:
...
 Special filters are some helpers to
 
+### or, and
+``or``, ``and`` extend query about extra condition, e.g.:
+```json
+{
+    "labels": "name",
+    "series": "serverroom__rack",
+    "excludes": {
+        "name__startswith|or": [null, ''],
```
Comment (ghost) [inline docs/user/dashboards.md]: `"name__startswith|or": [null, ''],` -> `"name__exact|or": [null, ''],`?

**Comment by szok** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**